src/request.js

```diff
@@ -178,13 +177,31 @@ Object.defineProperties(Request.prototype, {
 });
 
 /**
- * Get the AbortSignal object belonging to a Request.
+ * Indicates whether Request instance is listening for abort signal.
+ *
+ * @param   Request      A Request instance
+ * @return  boolean
+ */
+export function isListeningForAbortSignal(request) {
+	const signal = request.signal;
+	return signal !== undefined
+		&& signal.aborted !== true
+		&& request[INTERNALS].onSignalAbort !== undefined;
+}
+
+/**
+ * Removes the callback attached to a Request's signal "abort" event.
  *
  * @param   Request      A Request instance
- * @return  AbortSignal  request's signal
+ * @return  boolean      Indicates whether callback has been removed
  */
-export function getAbortSignal(request) {
```

Comment:
Since `signal` getter already exposes `signal`, I see no reason to have this method.

Reply:
We want to be safe against user overriding `signal` through `Object.defineProperty` or some other means.